src/world/Level.h

```diff
@@ -20,6 +22,7 @@ class Level {
 public:
 	std::unique_ptr<World> world;
 	const Content* const content;
+	std::list<Object*> objects;
 	Player* player;
```

Comment:
Раз уж ты меняешь логику создания player в классе Level, то можешь заодно изменить его тип на умный указатель, чтобы я у себя в пул реквесте это не делал, и чтобы меньше конфликтов было. Заодно можешь обновить метод `spawnObjectOfClass` чтобы возвращать `std::shared_ptr<T>`
```suggestion
	std::shared_ptr<Player> player;
```